packages/plugin-core/src/dendronExtensionInterface.ts

```diff
@@ -127,4 +132,5 @@ export interface IDendronExtension {
    * about registered Note Traits
    */
   get traitRegistrar(): NoteTraitService;
+  setEngine(svc: EngineAPIService): void;
```

Comment:
should be IEngineAPIService (if possible). 

Can we also move this up to be next to `getEngine`?